README.md

```diff
@@ -47,7 +47,7 @@ For complete API documentation, see the [Ably documentation](https://www.ably.co
 and require as:
 
 ```javascript
-var Ably = require('ably');
+var Ably = require('src/ably');
```

Comment:
I'm guessing the changes in this file are from an automatic find and replace, obviously we don't want to make a breaking change like this.

Reply:
Already fixed [here](https://github.com/ably/ably-js/pull/982/commits/f042f1cb73ddd4c122b807c8f712f875cdd26277)